sympy/combinatorics/tests/test_fp_groups.py

```diff
@@ -125,6 +125,413 @@ def test_coset_enumeration():
     # this test function contains the combined tests for the two strategies
     # i.e. HLT and Felsch strategies.
 
+    # Test for Issue #11352
+    F, a, b, c = free_group("a, b, c")
+    f = FpGroup(F, [a**11, b**5, c**4, (a*c)**3, b**2*c**-1*b**-1*c, a**4*b**-1*a**-1*b])
+    H = [c, b, c**2]
+    C_r = coset_enumeration_c(f, H)
+    C_r.compress(); C_r.standardize()
```

Comment:
@gxyd You want to me remove C_r = coset_enumeration_c(f,H) and the following line too ?